config/routes.rb

```diff
@@ -9,6 +9,8 @@
 
   get 'privacy_policy', to: 'policies#facebook'
 
+  resource :contacts, only: [:create]
```

Comment:
Hello!  I did create a new route solely for the purpose of testing the create action of the contacts controller in controller tests.  I could not successfully call the 

match "/contacts/:visit_id/:hosting_id" => "contacts#create", via: [:post]

route from rspec controller tests, despite much research and experimentation.  This route is here only for the use by rspec controller tests.